packages/docusaurus-theme-classic/src/theme/DocSidebar/index.tsx

```diff
@@ -64,6 +64,8 @@ function DocSidebarItemCategory({
     return isActive ? false : item.collapsed;
   });
 
+  const [enableCSSStyles, setEnableCSSStyles] = useState(true);
```

Comment:
Hi

You probably messed up with your Git workflow, because it's not supposed to be there  😅

Reply:
I know it was 2 am IST. I will fix it today, that is why I didn't message for a review 😅